PyRDF/backend/Backend.py

```diff
@@ -1,4 +1,10 @@
 class Backend(object):
+    """
+    A Generic Backend class (Interface)
+    that can be extended to define
+    specific 'execute()' functions
+
+    """
 
     def __init__(self, config={}):
```

Comment:
Docstring for both init and execute here

Reply:
Another comment, not for this PR, but for another one: there is the possibilty to specify abstract methods in Python, with the following syntax:
```
@abstractmethod
def execute(self, generator): pass
```
The advantage is that the check is not done at execution time like in the current solution. Can you verify that this works both in Python2 and Python3?

Reply:
@etejedor sure. I'll verify and make a new PR for this 👍